librosa/feature/utils.py

```diff
@@ -333,3 +333,93 @@ def sync(data, frames, aggregate=None, pad=True):
         start = end
 
     return data_agg
+
+
+@cache
+def chroma_to_tonnetz(chromagram, norm=np.inf):
```

Comment:
Can we refactor this?  I suggest the following:

* move the filterbank construction (`phi` in your variables) into the `filters` submodule
* move chroma_to_tonnetz into `feature.spectral`.  The `util` submodule is for generic feature manipulation, not specific mappings.